src/basilisp/cli.py

```diff
@@ -384,6 +411,9 @@ def _add_nrepl_server_subcommand(parser: argparse.ArgumentParser) -> None:
         default=".nrepl-port",
         help='the file path where the server port number is output to, defaults to ".nrepl-port".',
     )
+    _add_compiler_arg_group(parser)
```

Comment:
It is true that this group is missing from this subcommand, but adding the arguments alone is not sufficient to add the options to the nREPL server invocation. See `run` or `repl` implementation to see the plumbing required to actually use these.